mkosi/__init__.py

```diff
@@ -5755,7 +5762,12 @@ def print_summary(args: CommandLineArguments) -> None:
         "    Output nspawn Settings: "
         + none_to_na(args.output_nspawn_settings if args.nspawn_settings is not None else None)
     )  # NOQA: E501
-    MkosiPrinter.info("            Output SSH key: " + none_to_na(args.output_sshkey if args.ssh else None))
+    if args.ssh_private_key:
+        MkosiPrinter.info("           Private SSH key: " + args.ssh_private_key)
+        MkosiPrinter.info(f"            Public SSH key: {args.ssh_private_key}.pub")
+    else:
+        MkosiPrinter.info("            Output SSH key: " + none_to_na(args.output_sshkey if args.ssh else None))
+
```

Comment:
I think we can fold this into one
```suggestion
        MkosiPrinter.info("                   SSH key: " + none_to_na((args.ssh_private_key or args.output_sshkey) if args.ssh else None))

```
`or` is short-circuiting, so it will take the first truthty (i.e. non-`None`, non-empty) string

Reply:
This is definitely better, thanks.